### PR TITLE
feat: add new chains (fraxtal, linea, sei)

### PR DESCRIPTION
**Pull request by paulbalaji**
https://github.com/hyperlane-xyz/issues/issues/1279

paired with https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/3887

drive-by fix to `plumetestnet`, which was missing the `arbitrumnitro` info

### CODE REVIEW

File: chains/fraxtal/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api.fraxscan.com/api
+    family: etherscan
+    name: Fraxscan
+    url: https://fraxscan.com/
```
Comment (jmrossy) [inline chains/fraxtal/metadata.yaml:6]: Nit: no trailing slashes please

File: chains/linea/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api.lineascan.build/api
+    family: etherscan
+    name: LineaScan
+    url: https://lineascan.build/
```
Comment (jmrossy) [inline chains/linea/metadata.yaml:6]: ditto

**Review by jmrossy — COMMENTED** [review]
LGTM so far. don't forget a changeset.

**Review by nbayindirli — APPROVED** [review]
🚢

**Review by jmrossy — APPROVED** [review]

**Pull request merged**